client/src/Movies/Movie.js

```diff
@@ -25,43 +26,27 @@ export default class Movie extends Component {
         console.error(error);
       });
   };
-  // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+  //Uncomment this code when you're ready for the stretch problems
+  componentWillReceiveProps(newProps){
+    if(this.props.match.params.id !== newProps.match.params.id){
```

Comment:
I know this method was given to you from the assignment, but it's deprecated. Let's refactor this with CDU